### DISCUSSION TITLE
How can I hide the name badge of a hovered component?

### DISCUSSION THREAD

**Discussion by rozek** [GrapesJS/grapesjs#5340] [Q&A]
When hovering over a component view in the editor, a name badge is shown. In my case, the shown tag name is wrong:

<img width="246" alt="Name-Badge" src="https://github.com/GrapesJS/grapesjs/assets/514882/27cbfcbc-01f3-468e-9347-c3ccbf874b67">

How can I hide this badge (my model's defaults already contain `badgable:false,`) or correct the shown tag name, at least (my model's defaults also already contain a `tagName` and a `name` - and the latter is properly shown in the layers pane)?

**Comment by artf** [comment]
Looks like there is an issue with the badge re-rendering, it shows the previous name instead of hiding it.
I'll fix it in the next release.
